action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Install Shorebird'
+description: 'Installs shorebird'
+runs:
+  using: "composite"
+  steps:
+    - id: install
```

Comment:
Also nit but if we're not using the `id` for anything we can probably omit it

Reply:
Done.